.goreleaser.yml

```diff
@@ -13,7 +13,6 @@ dockers:
   image_templates:
   - "shopify/kubeaudit:latest"
   - "shopify/kubeaudit:{{ .Tag }}"
-  - "shopify/kubeaudit:v{{ .Major }}"
```

Comment:
so, by removing this line we guarantee that no breaking changes will be pushed automatically? Does that mean we will have to manually push those?

Reply:
Since we're on major version v0 it doesn't really make sense to have that image at all (you might as well use latest). The release instructions currently just don't say to push it so we've never had it available on dockerhub.